github/github.go

```diff
@@ -171,7 +128,8 @@ func getMilestones(baseURL string, token string, project string, state string) (
 	q.Set("state", state)
 	u.RawQuery = q.Encode()
 	newURL = u.String()
-	apiData, err := paginate(newURL, token)
+	api := "github"
```

Comment:
Unnecessary variable assignment.
It's better to pass the host directly in functions call since we just use it once.

Reply:
I'm not sure if that would work since the host could be something other than gitlab.com even though we're working with the GitLab API, for example.

Reply:
I meant we can pass the data that is inside ``api := "gitlab"`` directly to the Paginate function.
``utils.Paginate(u.String(), "gitlab", token)``

Reply:
Ah... yes, you're right. Thanks for clarifying! :)